### ISSUE TITLE
problème à la création d'un contenant

### ISSUE THREAD

**Issue by egauteur** [collec-science/collec-science#766]
Bonjour,
Je suis Elise Gauteur je travaille avec Agnès Starck, à lU3E de Rennes. J'essaie de créer un contenant sur l'instance :
https://u3e-rennes.collec-science.inrae.fr/
J'ai systématiquement ce message d’erreur 
"Une erreur indéterminée s'est produite pendant le traitement de la requête. Si le problème persiste, consultez l'administrateur de l'application"
Nadine n'y arrive pas non plus, elle obtient le même message et elle n'a rien vu dans les logs.

Merci, 
Elise Gauteur

**Comment by equinton** [comment]
Bizarre... Je viens de faire un test, j'ai bien pu créer un contenant, puis je l'ai supprimé, sans difficulté.
Il est très probable que ce soit lié à une limitation positionnée dans la base de données : il n'est actuellement pas possible de créer deux contenants avec le même nom.
![image](https://github.com/collec-science/collec-science/assets/6715861/a0f6e247-4ab0-43fa-9c43-380beb830d28)

Au besoin, je peux faire un point avec Nadine la semaine prochaine, pour discuter de ce sujet.

**Comment by egauteur** [comment]
Effectivement, je viens de vérifier si le contenant existait déjà et oui il existe ! Il était tout en bas de la liste (pas à la suite des autres) et je ne l'ai pas vu.
Désolée pour le dérangement 
Elise
